pages/bare_metal_cloud/dedicated_servers/rcw-changing-admin-password-on-windows/guide.en-gb.md

```diff
@@ -0,0 +1,166 @@
+
+> [!warning]
+>
+> This guide is not compatible with the `WinPE Rescue` mode.
+> Read [this guide](/pages/bare_metal_cloud/dedicated_servers/changing-admin-password-on-windows) if you are using the `WinPe Rescue` mode.
+>
+
+## Instructions
+
+### Step 1 - Rebooting the server into rescue mode <a name="step1"></a>
+
+The system has to be started in __*Windows customer rescue system*__ before the admin password can be changed.
+
+For detailed instructions, please refer to the [rescue mode guide](/pages/bare_metal_cloud/dedicated_servers/rescue-customer-windows).
+
+### Step 2 - Clearing the current password <a name="step2"></a>
+
+Connect to your server in Remote Desktop using the credentials provided by email.
+
+Please note that the user name for the rescue is `Administrator`.
+
+- If you server uses software RAID on your Windows disk, you have to import your Windows local disk to be able to access it: follow the instructions of the section [A) Importing your local Windows disk](#sectionA).
+- If you server does not use software RAID on your Windows disk, you should be able to access directly access the local Windows disk as explained in the section [B) Resetting the pasword ](#sectionB).
+
+#### A) Importing your local Windows disk <a name="sectionA"></a>
+
+##### 1. Access disk management
+
+Right-click the `Start`{.action} menu and select `Disk Management`{.action}.
+
+![disk_manager_menu](images/disk_manager_menu.png){.thumbnail}
+
+You can then see the disks and volumes of the server.
+
+![disk_manager_window](images/disk_manager_window1.png){.thumbnail}
+
+Your server's Windows disk is probably *disk 1*, so you must import it to be able to access it.
+Please note that if you have several disk groups, the number of the Windows disk may vary, you may have to import several disks to get the Windows one.
+
+##### 2. Importing local disk
+
+Right-click *Disk 1* and select `Online`{.action}.
+
+![disk_import_disk1](images/disk_manager_disk1on.png){.thumbnail}
+
+Do the same thing for the second disk (Disk 2) to properly import your software raid.
+
+Right-click *Disk 2* and select `Online`{.action}.
+
+![disk_import_disk2](images/disk_manager_disk2on.png){.thumbnail}
+
+Disks are now seen as "Dynamic" and "Foreign".
+
+Right-click *Disk 1* again and select `Import Foreign Disks`{.action}.
+
+![disk_import_menu](images/disk_manager_diskimport.png){.thumbnail}
+
+Click `OK` twice:
+
+![disk_import1](images/disk_import1.png){.thumbnail}
+
+![disk_import2](images/disk_import2.png){.thumbnail}
+
+Here, you can see that the local disk is now accessible and the Windows disk is drive `E:` (spreading on two disks because it is in software RAID).
+
+![disk_import_sync](images/disk_import_sync.png){.thumbnail}
+
+> [!primary]
+> In this example, the volume status is "Resynching" because the server was hard-rebooted in rescue mode. This is the normal behaviour and it is not caused by the rescue itself.
```

Comment:
```suggestion
> In this example the volume status is "Resynching" because the server was hard-rebooted in rescue mode. This is an expected status and it is not caused by the rescue itself.
```